externals/kyuubi-flink-sql-engine/pom.xml

```diff
@@ -139,6 +139,12 @@
       <artifactId>jul-to-slf4j</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.flink</groupId>
+      <artifactId>flink-test-utils_2.12</artifactId>
```

Comment:
Kyuubi uses log4j12, but this has transitive deps log4j2. We should exclude other log4j2 deps except to `log4j2-core`, and bridge it to sfl4j

Reply:
nit: `2.12` => `${scala_binary_version}`